src/figure/FigureStateActive.js

```diff
@@ -125,7 +127,7 @@ const initForm = ({ widget, formElement, formTpl, mediaEditor, togglePlaceholder
     );
 };
 
-export default function (stateFactory, ActiveState, formTpl, formElement, inlineHelper) {
+export default function (stateFactory, ActiveState, formTpl, formElement, inlineHelper, htmlEditor) {
```

Comment:
**quibble:** The signatures becomes heavier and heavier. Maybe it is the right time to think about simplifying it. If the API is not used in too many places, it could be simplified with a config object, allowing to addition of more entries without breaking the signature.

Reply:
Good point!